consensus/types/src/light_client_header.rs

```diff
@@ -53,6 +54,85 @@ pub struct LightClientHeader<E: EthSpec> {
     pub phantom_data: PhantomData<E>,
 }
 
+impl<E: EthSpec> LightClientHeader<E> {
+    pub fn block_to_light_client_header(
+        block: &SignedBeaconBlock<E>,
+        fork_name: ForkName,
```

Comment:
nit: we can probably just match on the `SignedBeaconBlock` variant without requiring a `fork_name`